lib/config/validation.ts

```diff
@@ -83,6 +83,10 @@ function isManagerPath(parentPath: string): boolean {
   );
 }
 
+function isBumpVersionPath(parentPath: string): boolean {
+  return regEx(/^bumpVersions\[\d+]$/).test(parentPath);
+}
```

Comment:
Please ensure this gets covered in a test, because the current implementation is too narrow.